internal/app/go-filecoin/internal/submodule/storage_mining_submodule.go

```diff
@@ -38,53 +48,57 @@ func NewStorageMiningSubmodule(minerAddr, workerAddr address.Address, ds datasto
 
 	smbe := piecemanager.NewStorageMinerBackEnd(storageMiner, s)
 	sbbe := postgenerator.NewSectorBuilderBackEnd(s)
+	stateViewer := state.NewViewer(b.CborStore)
 
 	modu := &StorageMiningSubmodule{
 		PieceManager:  smbe,
 		PoStGenerator: sbbe,
+		minerNode:     minerNode,
```

Comment:
IIRC @laser left dependence on the concrete structs backing the interfaces off of the submodule so that we could more easily fake the proving/sealing system during test setup. I'm ok leaving it if you think its better but if you haven't already consider how this impacts testability.